### PR TITLE
Update VS code Camel K to 0.0.22 eclipse/che#19360

### PR DESCRIPTION
**Pull request by apupier**
to be tested

### What does this PR do?

Update VS code Camel K to 0.0.22 

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: che-theia-plugins.yaml [hunk 1]
```diff
@@ -200,11 +200,13 @@ plugins:
...
     extensions:
       - >-
-        https://download.jboss.org/jbosstools/vscode/stable/vscode-camelk/vscode-camelk-0.0.21-109.vsix
+        https://download.jboss.org/jbosstools/vscode/stable/vscode-camelk/vscode-camelk-0.0.22-135.vsix
       - >-
         https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.2.1.vsix
       - >-
         https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.11.1.vsix
+      - >-
+        https://open-vsx.org/api/redhat/vscode-commons/0.0.4/file/redhat.vscode-commons-0.0.4.vsix
```
Comment (apupier) [inline che-theia-plugins.yaml:209]: not available on download.jboss.org

**Comment by apupier** [comment]
error doesn't seem to be related to this PR:
https://github.com/eclipse/che-plugin-registry/pull/894/checks?check_run_id=2169753031#step:7:22
```
Error response from daemon: Head https://quay.io/v2/eclipse/che-plugin-registry/manifests/nightly: received unexpected HTTP status: 502 Bad Gateway
Error: Process completed with exit code 1.
```

**Comment by benoitf** [comment]
https://status.quay.io/

**Comment by apupier** [comment]
tested that VS Code extension with devfile is starting fine using thsi devfile:

```
apiVersion: 1.0.0
metadata:
  name: apache-camel-k-n23kt
projects:
  - name: camel-k-examples
    source:
      location: 'https://codeready-codeready-workspaces-operator.apps.sandbox-m2.ll9k.p1.openshiftapps.com/devfile-registry/resources/apache-camel-k-camel-k-examples-master.zip'
      type: zip
components:
  - id: redhat/vscode-xml/latest
    type: chePlugin
    registryUrl: 'https://apupier-che-plugin-registry-dev-39sap.surge.sh/v3'
  - id: redhat/vscode-apache-camel/latest
    type: chePlugin
    registryUrl: 'https://apupier-che-plugin-registry-dev-39sap.surge.sh/v3'
  - id: redhat/vscode-camelk/latest
    type: chePlugin
    registryUrl: 'https://apupier-che-plugin-registry-dev-39sap.surge.sh/v3'
    alias: vscode-camelk
```

File: che-theia-plugins.yaml (hunk 1 above)
Comment (svor) [reply che-theia-plugins.yaml:209]: it is available on snapshots https://download.jboss.org/jbosstools/snapshots/vscode-commons//vscode-commons-0.0.4-44.vsix

**Review by svor — APPROVED** [review]

**Review by ericwill — APPROVED** [review]

**Pull request merged**